Slim/Formats/Playlists/CUE.pm

```diff
@@ -25,6 +27,33 @@ use Slim::Utils::Unicode;
 my $log = logger('formats.playlists');
 my $prefs = preferences('server');
 
+#List of the standard cuesheet commands.
+# see http://en.wikipedia.org/wiki/Cue_sheet_%28computing%29
+#     http://wiki.hydrogenaud.io/index.php?title=Cue_sheet
+#
+# here as an alternate source of CDRWIN help: 
+# http://digitalx.org/cue-sheet/syntax/
+#
+my @standardCueCommands = qw(CATALOG CDTEXTFILE FILE FLAGS INDEX ISRC PERFORMER
+                             POSTGAP PREGAP REM SONGWRITER TITLE TRACK);
+
```

Comment:
I'd suggest you replace @standardCueCommands and @refusedCueCommands with hashes. These should be much faster to look up further down the code. And more elegant, too ;-)

```perl
my %standardCueCommands = (
   CATALOG => 1,
   CDTEXTFILE => 1,
   ...
);
```

Then, instead of looping through the list over and over again, you can simply:

```perl
sub _isStandardCommand {
   return $standardCueCommands{shift};
}
```

...or simply check the hash value instead of calling _isStandardComand.